hoot-core/src/main/cpp/hoot/core/io/ElementStreamer.cpp

```diff
@@ -228,6 +228,7 @@ void ElementStreamer::stream(const QStringList& inputs, const QString& out,
     std::dynamic_pointer_cast<ElementOutputStream>(writer);
   std::shared_ptr<PartialOsmMapWriter> partialWriter =
     std::dynamic_pointer_cast<PartialOsmMapWriter>(writer);
+  partialWriter->initializePartial();
```

Comment:
Streaming files has to explicitly call `initializePartial()` to get the header written since it isn't in `open()` anymore.